index.html

```diff
@@ -313,6 +313,22 @@ <h2 class="primary-text-about-me">About
     </section>
     <!--End-About-me-->
   </main>
+  <footer>
+    <div class="contact-me">
+      <form class="form-contact-me" action="#" method="post">
+        <h2 class="primary-text-contact-me"> Contact me</h2>
+        <p class="secondary-text-contact">If you have an application you are interested in developing, a feature that
+          you need built or a project that needs coding. I’d love to help with</p>
+        <input class="input" name="name" id="name" type="name" placeholder="Enter your name" pattern="[A-Za-z/ ]{0,30}" required>
+        <input class="input" name="Email" id="email" type="email" placeholder="Enter your email" required>
+        <textarea class="input" name="message" id="message" placeholder="Write your message here" maxlength="500"></textarea>
```

Comment:
:red_circle: Kindly make sure this field has a required attribute to make sure it is filled out before submitting the form.

Reply:
> Correction

*Attribute* `required` *added to* `<textarea>`.